docs/proposals/04-new-core-gardener-cloud-apis.md

```diff
@@ -282,6 +283,41 @@ status:
   observedGeneration: 1
 ```
 
+### `Project` resource
+
+Special note: The `members` and `viewers` field of the `garden.sapcloud.io/v1beta1.Project` resource will be merged together into one `members` field.
+Every member will have a role that is either `admin` or `viewer`.
+This will allow us to add new roles without changing the API.
+
+```yaml
+apiVersion: garden.sapcloud.io/v1beta1
+kind: Project
+metadata:
+  annotations:
+    project.garden.sapcloud.io/namespace: garden-core
```

Comment:
#### question

do we need this annotation for the new API?
If yes, I am confused because the project name is `example` and `spec.namespace` is `garden-example`, but the annotation value is `garden-core`.

Reply:
No, you're right, thanks, I remove it.